domains/malwaremusashi.js

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MalwareMusashi",
+        "email": "You may contact me on X at @M4lw4r3Mus4sh1"
```

Comment:
```suggestion
        "username": "MalwareMusashi"
```

remove email,

you can add x if you want

```
        "X": "M4lw4r3Mus4sh1"
```